tests/test_inverse_kinematics.py

```diff
@@ -0,0 +1,218 @@
+"""Tests numerical inverse kinematics pipeline.
+"""
+import unittest
+from colony_picker.inverse_kinematics import*
+from tests.helper_functions_for_tests import*
+import random
+import math
+import warnings
+
+ar3_alpha_vals = [-(math.pi/2), 0, math.pi/2, -(math.pi/2), math.pi/2, 0]
+ar3_a_vals = [64.2, 305, 0, 0, 0, 0]
+ar3_d_vals = [169.77, 0, 0, -222.63, 0, -36.25]
+ar3_theta_offsets = [0, 0, -math.pi/2, 0, 0, math.pi]
+ar3_num_joints = len(ar3_alpha_vals)
+
+ar3_dh_params = np.array(
+    [ar3_theta_offsets, ar3_alpha_vals, ar3_a_vals, ar3_d_vals])
+
+animation_test_warning = "Only one animation test should be run at a time."
+
+
+class TestInverseKinematics(unittest.TestCase):
+    """A unit test harness for testing the numerical inverse kinematics pipeline
+    """
+    @classmethod
+    def setUpClass(cls):
```

Comment:
If you name this `setUp` then it will automatically be called before all of your tests.